src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import url from 'url';
+import {Container, Nav, Navbar} from "react-bootstrap";
+import {Link, NavLink} from "react-router-dom";
+import "./styles/NavigationBar.css";
+
+const logo = url.resolve(process.env.PUBLIC_URL, "/icons/se-logo-trimmed.png");
+
+const NavigationBar = () => (
+  <Navbar bg="transparent" expand="sm" collapseOnSelect >
```

Comment:
Let's update the navbar items to be: `Home`, `About`, `Events` (in this order & capitalized)